src/lib.rs

```diff
@@ -110,11 +116,13 @@ pub enum Instruction {
 impl rlp::Encodable for Instruction {
     fn rlp_append(&self, s: &mut rlp::RlpStream) {
         match self {
-            Instruction::EXTENSION(ref ext) => s.begin_list(2).append(&3usize).append_list(ext),
-            Instruction::BRANCH(size) => s.begin_list(2).append(&0usize).append(size),
-            Instruction::HASHER(size) => s.begin_list(2).append(&1usize).append(size),
-            Instruction::LEAF(size) => s.begin_list(2).append(&2usize).append(size),
-            Instruction::ADD(index) => s.begin_list(2).append(&4usize).append(index),
+            Instruction::EXTENSION(ref ext) => {
```

Comment:
Linter fail? Let's not add curly braces for just one line

Reply:
Well you need to disable `cargo fmt` then.